app/views/partials/_articles.html.erb

```diff
@@ -0,0 +1,69 @@
+<% total_articles = total_articles ||= 6 %>
+
+<% @article_types = ArticleType.all %>
```

Comment:
No reason, just not used to using instance variables so forgot about them! Whats the reason for using instance variables over local ones here?